Tests/govuk_ios/govuk_ios_unit_tests/Specs/Coordinators/SettingsCoordinatorTests.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import XCTest
+
+@testable import govuk_ios
+
+class SettingsCoordinatorTests: XCTestCase {
+    
+    override class func setUp() {
+        super.setUp()
+        UIView.setAnimationsEnabled(false)
+    }
+
+    @MainActor
+    func test_start_showsSettingsViewController() {
```

Comment:
suggest rename to "setsSettingsViewController" because its specific to what is being done.